pandas/io/gbq.py

```diff
@@ -106,61 +31,18 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         **kwargs)
 
 
-def to_gbq(dataframe, destination_table, project_id, chunksize=10000,
-           verbose=True, reauth=False, if_exists='fail', private_key=None):
-    """Write a DataFrame to a Google BigQuery table.
-
-    THIS IS AN EXPERIMENTAL LIBRARY
-
-    The main method a user calls to export pandas DataFrame contents to
-    Google BigQuery table.
-
-    Google BigQuery API Client Library v2 for Python is used.
-    Documentation is available at
-    https://developers.google.com/api-client-library/python/apis/bigquery/v2
-
-    Authentication to the Google BigQuery service is via OAuth 2.0.
+read_gbq.__doc__ = generate_dynamic_docstring(
+    lambda: _try_import().read_gbq.__doc__)
```

Comment:
It would be great to start testing that these changes don't damage the serializability of Pandas functions.  Pandas does a lot of dynamic things to its functions and methods which has caused some pain in the past.